nbviewer/utils.py

```diff
@@ -5,7 +5,14 @@
 #  the file COPYING, distributed as part of this software.
 #-----------------------------------------------------------------------------
 
-import base64
+# https://docs.python.org/3.1/library/base64.html#base64.decodestring
+try:
+    from base64 import encodebytes as encodestring
+    from base64 import decodebytes as decodestring
+except ImportError:
+    from base64 import encodestring
```

Comment:
probably makes more sense to import `deprecated as new` than `new as deprecated`, and update below.